vms/vm-under-test/scripts/customize-vm

```diff
@@ -90,21 +90,24 @@ cat <<'EOF' > "$checkup_boot_script_full_path"
 # Copyright 2024 Red Hat, Inc.
 #
 
-set -e
```

Comment:
Commit message:
> vm-under-test/customize-vm: Add flag to run tuned-adm once

This line repeats itself twice.

Reply:
DONE